packages/client/test/integration/fullsync.spec.ts

```diff
@@ -40,17 +41,35 @@ tape('[Integration:FullSync]', async (t) => {
       minPeers: 2,
     })
     await localService.synchronizer.stop()
-    await localServer.discover('remotePeer1', '127.0.0.2')
+    const peer = await localServer.discover('remotePeer1', '127.0.0.2')
     await localServer.discover('remotePeer2', '127.0.0.3')
     localService.config.events.on(Event.SYNC_SYNCHRONIZED, async () => {
-      if (localService.chain.blocks.height.toNumber() === 10) {
-        t.pass('synced with best peer')
-        await destroy(localServer, localService)
-        await destroy(remoteServer1, remoteService1)
-        await destroy(remoteServer2, remoteService2)
-        t.end()
+      switch (localService.chain.blocks.height.toNumber()) {
```

Comment:
i'm kind of confused what you changed here

Reply:
My intent was just to have a complete integration test of the full sync process where it syncs up to the tip of the chain and then inserts a new block after it's announced by a peer.  It felt like a logical extension of this particular test though if you think that should be in a separate integration test, I'm good with that too.  I'll work on adding some more unit tests as well.